Exercise3/Models/Ultilities.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Exercise3.Models
+{
+    public class Ultilities
+    {
+        public static List<ManagerItem> GetFileAndFolderNames(string path)
+        {
+            var mList = new List<ManagerItem>();
+            var fileAndFolderNames = Directory.GetFileSystemEntries(path);
+            foreach (var name in fileAndFolderNames)
+            {
+                if (IsFolder(name))
+                {
+                    mList.Add(new ManagerItem(Resource.Drawable.folder32, name));
+                }
+                else if (IsMediaFile(name))
+                {
+                    mList.Add(new ManagerItem(Resource.Drawable.videofile32, name));
+                }
+                else if (IsImage(name))
+                {
+                    mList.Add(new ManagerItem(Resource.Drawable.image32, name));
+                }
+                else
+                {
+                    mList.Add(new ManagerItem(Resource.Drawable.file32, name));
+                }
+            }
+            return mList;
+        }
+
+        private static bool IsImage(string name)
+        {
+            var imageTag = new[] { ".png", ".jpg", ".gif" };
+            return imageTag.Any(name.Contains);
```

Comment:
Need to change
- Replace `Contains` to `EndWith`
- if the file name is abc.PNG, this function will not work well.

Reply:
done